pkg/cmd/server/kubernetes/node.go

```diff
@@ -280,39 +283,43 @@ func (c *NodeConfig) RunServiceStores(enableProxy, enableDNS bool) {
 
 // RunKubelet starts the Kubelet.
 func (c *NodeConfig) RunKubelet() {
-	if c.KubeletConfig.ClusterDNS == nil {
+	var clusterDNS net.IP
+	if c.KubeletServer.ClusterDNS == "" {
```

Comment:
I get tired sometimes.

On Sep 14, 2016, at 1:06 PM, David Eads <notifications@github.com> wrote:

In pkg/cmd/server/kubernetes/node.go
<https://github.com/openshift/origin/pull/10777#discussion_r78791071>:

> @@ -280,39 +283,43 @@ func (c *NodeConfig) RunServiceStores(enableProxy, enableDNS bool) {
>
>  // RunKubelet starts the Kubelet.
>  func (c *NodeConfig) RunKubelet() {
> -	if c.KubeletConfig.ClusterDNS == nil {
> +	var clusterDNS net.IP
> +	if c.KubeletServer.ClusterDNS == "" {

hahahahaha. Really? len(c.KubeletServer.ClusterDNS). That's funny from you.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/10777/files/ba9a06d1d23905e3c92cd9aeb98c56290c321f0d..a5c6b78ddd75c9f54dedcc94bd1296c0ac25359b#r78791071>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ABG_p50gCNyi_XpVSx0t-EQV-xjvW8uPks5qqCmIgaJpZM4JzJrj>
.